xwiki-platform-distribution/pom.xml

```diff
@@ -379,23 +379,6 @@
                       <class>org.w3c.dom.events.UIEvent</class>
                     </classes>
                   </exception>
-                  <!-- This is a PITA. The classes are actually incompatible! We should do something about it but I
```

Comment:
Unfortunately css4j is not the only thing in the XWiki stack which depends on sac as far as I can see so I don't think we can remove that.

Reply:
That's what I was fearing (I was unable to verify the build). Removed that patch, thank you.